src/content/docs/infrastructure/host-integrations/host-integrations-list/haproxy-monitoring-integration.mdx

```diff
@@ -10,52 +10,47 @@ redirects:
   - /docs/haproxy-integration-new-relic-infrastructure
 ---
 
-Our HAProxy [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) collects and sends inventory and metrics from your HAProxy instance to the New Relic platform, where you can aggregate and visualize key performance metrics for the HAProxy server, frontends, and backends.
+Our HAProxy [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) collects and sends inventory and metrics from your HAProxy instance to the New Relic platform, where you can aggregate and visualize key performance metrics for the HAProxy server, frontends, and backends.   
```

Comment:
this is a bit of a mouthful. what does 'collects and sends inventory' mean? Like the contents of an ecommerce inventory? 

Also might break this sentence up to be clearer.